### DISCUSSION TITLE
Build Plus Toolkit in Ubuntu 20.04.3 LTS

### DISCUSSION THREAD

**Discussion by lbo2021** [PlusToolkit/PlusLib#858] [Q&A]
Hi,

I’m trying to build Plus Toolkit 2.8 in Ubuntu 20.04.3 LTS.
I have followed the information of this page https://github.com/PlusToolkit/PlusBuild/blob/master/Docs/BuildInstructionsLinux.md

The only thing I did in a different way is that I launched this command “sudo apt build-dep vtk7” instead of “sudo apt build-dep vtk” because the second one is not working.

During the building process, I am getting the following error:

```
[100%] Linking CXX shared library /home/user/devel/PlusBuild-bin/bin/libvtkViewsQt-7.1.so
[100%] Built target vtkViewsQt
[ 51%] No install step for 'vtk'
[ 53%] Completed 'vtk'
[ 53%] Built target vtk
make: *** [Makefile:95: all] Error 2
```

Have any of you already experienced this issue?
Do you have any suggestion?

Thank you

**Comment by lassoan** [comment]
Plus supports "v8.2.0" "v9.0.0" "v9.0.1" VTK versions. - see https://github.com/PlusToolkit/PlusBuild/blob/master/SuperBuild/External_VTK.cmake#L49

What VTK version is installed if you run `sudo apt build-dep vtk`?

**Reply by lbo2021** [reply]
Thanks for your reply.

No version is installed because `sudo apt build-dep vtk` returns `E: Unable to find a source package for vtk`

`sudo apt build-dep vtk7` instead probably installs something else which is not VTK

**Comment by adamrankin** [comment]
Should we remove that instruction and let it be built just like other OS's?

**Reply by lassoan** [reply]
I agree. The build instruction should explain how to use PlusBuild to download and build PlusLib and all dependencies.

**Reply by lbo2021** [reply]
So can you please tell me how can I avoid the error I mentioned above?

**Reply by lassoan** [reply]
Something like this should build Plus:

```
sudo apt update && sudo apt install git build-essential cmake qt5-default qtmultimedia5-dev qttools5-dev libqt5xmlpatterns5-dev qtbase5-private-dev libqt5x11extras5-dev libxt-dev
git clone git://github.com/PlusToolkit/PlusBuild.git
mkdir PlusBuild-bin
cd PlusBuild-bin
cmake ../PlusBuild
make -j8
```

**Reply by lbo2021** [reply]
Thank you. It worked
